src/kawaz/core/personas/migrations/0005_auto_20161002_2154.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2016-10-02 12:54
```

Comment:
**Suggestion** 
Why don't you write the reason of the migration. Let's make a new habit for the future.
Note that this is just a suggestion so you can ignore if you disagree the idea or too lazy.